packages/autorest.python/samples/specification/multiapi/generated/azure/multiapi/sample/v1/operations/_multiapi_service_client_operations.py

```diff
@@ -200,8 +200,8 @@ def _test_lro_initial(
         cls: ClsType[Optional[_models.Product]] = kwargs.pop("cls", None)
 
         content_type = content_type or "application/json"
-        _json = None
-        _content = None
+        _json: Any = None
```

Comment:
i think json should be typed with JSONType and content with IO type

Reply:
Adding `Any` is the easiest way to avoid mypy error:

Reply:
trying to go through the generated code diffs is v difficult, i think we should remove this change: if it's pasing mypy now i'm not sure why you need this change. if we really. want this we can add it in another pr for better readability

Reply:
have added specific logic to avoid influencing legacy code.